### PR TITLE
FUSETOOLS-2171 - upgrade to Neon TP 4.4.0.CR1

### PR DESCRIPTION
**Pull request by lhein**

### CODE REVIEW

File: eclipse/site/pom.xml [hunk 1]
```diff
@@ -171,7 +171,7 @@
       <plugin>
         <groupId>org.jboss.tools.tycho-plugins</groupId>
         <artifactId>repository-utils</artifactId>
-        <version>${jbossTychoPluginsVersion}</version>
+        <version>0.23.1</version>
```
Comment (apupier) [inline eclipse/site/pom.xml]: we might use jboss-tycho-version property, it is the one used in Fuse Tooling

Comment (lhein) [inline eclipse/site/pom.xml]: sadly there seems to be no repository-utils in that version available. At least my builds fail locally.

Comment (lhein) [inline eclipse/site/pom.xml]: latest non-snapshot is 0.23.3

Comment (apupier) [inline eclipse/site/pom.xml]: the other Tycho version are 0.26.0, in Fuse Tooling we are using 0.26.0-SNAPSHOT
Why are we still in 0.23.1?

Comment (lhein) [inline eclipse/site/pom.xml]: tbh I have no clue. We can try to switch to 0.26.0 later but that is out of scope of this PR. The whole Maven Reactor of the SY tools is messed up and should be cleaned up soon.

Comment (apupier) [inline eclipse/site/pom.xml]: The jboss-tycho-version is currently set to 0.26.0, the creation of the TP is nto possible, we need to use 0.26.0-SNAPSHOT, I asked on hipchat if the tag is planned.
My recommendation until then:
- use jboss-tycho-version for repository-utils too
- set property value to 0.26.0-SNAPSHOT

**Comment by apupier** [comment]
testPR

**Comment by apupier** [comment]
`05:42:32 [ERROR] An error occurred while transferring artifact canonical: osgi.bundle,org.fusesource.ide.camel.model.service.v2_17_3,9.0.0.CI-v20161019-0741-B36 from repository http://download.jboss.org/jbosstools/neon/staging/builds/integration-stack/fuse-tooling/9.0.0.CI:
05:42:32 [ERROR]    Artifact not found: http://download.jboss.org/jbosstools/neon/staging/builds/integration-stack/fuse-tooling/9.0.0.CI/plugins/org.fusesource.ide.camel.model.service.v2_17_3_9.0.0.CI-v20161019-0741-B36.jar.
05:42:32 [ERROR] Internal error: org.eclipse.tycho.repository.local.MirroringArtifactProvider$MirroringFailedException: Could not mirror artifact osgi.bundle,org.fusesource.ide.camel.model.service.v2_17_3,9.0.0.CI-v20161019-0741-B36 into the local Maven repository.See log output for details. http://download.jboss.org/jbosstools/neon/staging/builds/integration-stack/fuse-tooling/9.0.0.CI/plugins/org.fusesource.ide.camel.model.service.v2_17_3_9.0.0.CI-v20161019-0741-B36.jar -> [Help 1]
05:42:32 org.apache.maven.InternalErrorException: Internal error: org.eclipse.tycho.repository.local.MirroringArtifactProvider$MirroringFailedException: Could not mirror artifact osgi.bundle,org.fusesource.ide.camel.model.service.v2_17_3,9.0.0.CI-v20161019-0741-B36 into the local Maven repository.See log output for details.
05:42:32 	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:121)
05:42:32 	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
05:42:32 	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
05:42:32 	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
05:42:32 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
05:42:32 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
05:42:32 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
05:42:32 	at java.lang.reflect.Method.invoke(Method.java:498)
05:42:32 	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
05:42:32 	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
05:42:32 	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
05:42:32 	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
05:42:32 Caused by: org.eclipse.tycho.repository.local.MirroringArtifactProvider$MirroringFailedException: Could not mirror artifact osgi.bundle,org.fusesource.ide.camel.model.service.v2_17_3,9.0.0.CI-v20161019-0741-B36 into the local Maven repository.See log output for details.
05:42:32 	at org.eclipse.tycho.repository.local.MirroringArtifactProvider.downloadArtifact(MirroringArtifactProvider.java:226)
05:42:32 	at org.eclipse.tycho.repository.local.MirroringArtifactProvider.makeOneFormatLocallyAvailable(MirroringArtifactProvider.java:211)
05:42:32 	at org.eclipse.tycho.repository.local.MirroringArtifactProvider.makeLocallyAvailable(MirroringArtifactProvider.java:182)
05:42:32 	at org.eclipse.tycho.repository.local.MirroringArtifactProvider.getArtifactFile(MirroringArtifactProvider.java:121)
05:42:32 	at org.eclipse.tycho.repository.p2base.artifact.provider.CompositeArtifactProvider.getArtifactFile(CompositeArtifactProvider.java:85)
05:42:32 	at org.eclipse.tycho.p2.target.TargetPlatformBaseImpl.getLocalArtifactFile(TargetPlatformBaseImpl.java:128)
05:42:32 	at org.eclipse.tycho.p2.resolver.P2ResolverImpl.addUnit(P2ResolverImpl.java:254)
05:42:32 	at org.eclipse.tycho.p2.resolver.P2ResolverImpl.toResolutionResult(P2ResolverImpl.java:217)
05:42:32 	at org.eclipse.tycho.p2.resolver.P2ResolverImpl.resolveDependencies(P2ResolverImpl.java:209)
05:42:32 	at org.eclipse.tycho.p2.resolver.P2ResolverImpl.resolveDependencies(P2ResolverImpl.java:116)
05:42:32 	at org.eclipse.tycho.p2.resolver.P2DependencyResolver.doResolveDependencies(P2DependencyResolver.java:367)
05:42:32 	at org.eclipse.tycho.p2.resolver.P2DependencyResolver.resolveDependencies(P2DependencyResolver.java:335)
05:42:32 	at org.eclipse.tycho.core.resolver.DefaultTychoResolver.resolveProject(DefaultTychoResolver.java:117)
05:42:32 	at org.eclipse.tycho.core.maven.TychoMavenLifecycleParticipant.afterProjectsRead(TychoMavenLifecycleParticipant.java:95)
05:42:32 	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:266)
05:42:32 	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
05:42:32 	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
05:42:32 	... 11 more
05:42:32 Caused by: java.io.FileNotFoundException: http://download.jboss.org/jbosstools/neon/staging/builds/integration-stack/fuse-tooling/9.0.0.CI/plugins/org.fusesource.ide.camel.model.service.v2_17_3_9.0.0.CI-v20161019-0741-B36.jar
05:42:32 	at org.eclipse.equinox.internal.p2.transport.ecf.RepositoryStatusHelper.checkFileNotFound(RepositoryStatusHelper.java:297)
05:42:32 	at org.eclipse.equinox.internal.p2.transport.ecf.FileReader.checkException(FileReader.java:478)
05:42:32 	at org.eclipse.equinox.internal.p2.transport.ecf.FileReader.sendRetrieveRequest(FileReader.java:435)
05:42:32 	at org.eclipse.equinox.internal.p2.transport.ecf.FileReader.readInto(FileReader.java:358)
05:42:32 	at org.eclipse.equinox.internal.p2.transport.ecf.RepositoryTransport.download(RepositoryTransport.java:101)
05:42:32 	at org.eclipse.equinox.internal.p2.transport.ecf.RepositoryTransport.download(RepositoryTransport.java:156)`
the build failed seems caused by an error on the update site

**Pull request merged**